gdk/PixbufLoader.custom

```diff
@@ -141,3 +147,30 @@
 		{
 			return new PixbufLoader (System.Reflection.Assembly.GetCallingAssembly (), resource);
 		}
+
+        class ByteArrayPool
+        {
+            WeakReference<System.Collections.Concurrent.ConcurrentQueue<byte[]>> queuedBuffers =
```

Comment:
I don't want to keep this in memory all the time, and it's a queue because _some_ users of this library can load pixbufs from multiple background threads.